tests/playwright/utils/deploy_utils.py

```diff
@@ -128,6 +129,18 @@ def write_requirements_txt(app_dir: str) -> None:
         f.write(f"git+https://github.com/posit-dev/py-shiny.git@{git_hash}\n")
 
 
+def assert_rsconnect_file_updated(file_path: str, max_minutes: int) -> None:
+    """
+    Asserts that the specified file has been updated within the last `max_minutes` minutes.
+    """
+    current_time = time.time()
+    mod_time = os.path.getmtime(file_path)
+    time_diff = (current_time - mod_time) / 60
+    assert (
+        time_diff < max_minutes
+    ), f"File '{file_path}' was not updated within the last {max_minutes} minutes which means the deployment failed or was skipped"
```

Comment:
```suggestion
def assert_rsconnect_file_updated(file_path: str, min_mtime: float) -> None:
    """
    Asserts that the specified file has been updated since `min_mtime` (seconds since epoch).
    """
    mtime = os.path.getmtime(file_path)
    time_diff = (mtime - min_mtime)
    assert (
        mtime > min_mtime
    ), f"File '{file_path}' was not updated during app deployment which means the deployment failed"
```